### PR TITLE
Update Sentiment Analysis for .10

### LINKED ISSUE #10340: Did the API change?
I followed all the steps but am having trouble getting the code to compile. I am targeting netcoreapp2.1 and

1. mlContext.Data.CreateTextReader should be mlContext.Data.CreateTextLoader I think.

2. mlContext.CreateStreamingDataView should be mlContext.Data.ReadFromEnumerable?

3. I can not get the following to work at all:

            var predictedResults = predictions.AsEnumerable<SentimentPrediction>(mlContext, reuseRowObject: false);

There does not seem to be an AsEnumerable (extension) method.

---
#### Documentdetails

⚠ *Dit gedeelte niet bewerken. Het is vereist om problemen te koppelen tussen docs.microsoft.com en GitHub.*

* ID: c8e22825-c4ee-5a36-f911-8ad456970ecc
* Version Independent ID: ad0bf640-7dfc-1184-077c-358bb14277a7
* Content: [Use ML.NET in a sentiment analysis binary classification scenario](https://docs.microsoft.com/nl-nl/dotnet/machine-learning/tutorials/sentiment-analysis)
* Content Source: [docs/machine-learning/tutorials/sentiment-analysis.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/sentiment-analysis.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### LINKED ISSUE #10390: Update for v0.10
This line is broken in v0.10 as the constructors have been made internal:
var trainer = new SdcaMultiClassTrainer(_mlContext,DefaultColumnNames.Label, DefaultColumnNames.Features);

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77fc5686-2288-d607-8136-3146ed02dbf6
* Version Independent ID: 8a16b9e9-b5cb-a55e-2135-191bc62c1656
* Content: [Use ML.NET in a GitHub issue multiclass classification scenario](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification)
* Content Source: [docs/machine-learning/tutorials/github-issue-classification.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/tutorials/github-issue-classification.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

### PR DESCRIPTION
**Pull request by JRAlexander**
Update Sentiment Analysis Tutorial for .10 
Updated samples is at https://github.com/dotnet/samples/pull/632. 

Fixes #10340

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-10419)

### CODE REVIEW

**Review by luisquintanilla — CHANGES_REQUESTED** [review]
Looks good. Minor changes.

**Comment by JRAlexander** [comment]
Thank you for the review, @luisquintanilla!! I believe I've addressed your concerns.

**Review by luisquintanilla — APPROVED** [review]
Thanks for the changes @JRAlexander . Okay to merge.
